.github/workflows/slurm_packages.yml

```diff
@@ -43,14 +47,15 @@ jobs:
       - name: Build docker images
         uses: docker/build-push-action@14487ce63c7a62a4a324b0bfb37086795e31c6c1 # v6.16.0
         with:
-          file: ${{ matrix.image.file }}
-          push: ${{ matrix.image.push }}
+          file: slurm-packages/Dockerfile
+          push: false
           tags: slurm_packages:${{ matrix.slurm.version }}-${{ env.SHORT_SHA }}
-          platforms: ${{ matrix.image.platforms }}
-          load: ${{ matrix.image.load }}
+          platforms: ${{ matrix.platform }}
+          load: true
           build-args: |
-            ${{ matrix.image.build_args }}
+            BASE_IMAGE=cr.eu-north1.nebius.cloud/soperator/nvidia/cuda:12.4.1-cudnn-devel-ubuntu22.04
```

Comment:
Doesn't this image also have to be platform-specific?

Reply:
Yes, its specification is on the 53 line: `platforms: ${{ matrix.platform }}`

https://docs.docker.com/build/ci/github-actions/multi-platform/